README.md

```diff
@@ -1 +1,76 @@
-# AndroidModularizeStarter
+# AndroidModularizeStarter
+
+This is a simple news headline reading app with dummy user system to demonstrate how to modularize an Android application
+
```

Comment:
I would also provide the scenario we faced, and why this makes sense lead to this project, to support the next section, `Why modularizing`